packages/devtools_app/lib/src/shared/banner_messages.dart

```diff
@@ -479,6 +479,67 @@ For the most accurate absolute memory stats, relaunch your application in ''',
   }
 }
 
+class DebuggerIdeRecommendationMessage {
+  const DebuggerIdeRecommendationMessage(this.screenId);
+
+  final String screenId;
+
+  BannerMessage build(BuildContext context) {
+    final codeType =
+        (serviceConnection.serviceManager.connectedApp?.isFlutterAppNow ??
+                false)
+            ? 'Flutter'
+            : 'Dart';
+    final g3FlutterInstructionsLink =
+        devToolsExtensionPoints.g3FlutterIdeRecommendationLink();
```

Comment:
as a yak shave, can we rename the `devToolsExtensionPoints` global to match the class name (`devToolsEnvironmentParameters`)?

Reply:
Done